cves/CVE-2013-2251.yml

```diff
@@ -45,7 +45,9 @@ description_instructions: |
   that outsiders to Struts would not understand. Technology like "regular
   expressions" is fine, and security phrases like "invalid write" are fine to
   keep too.
-description:
+description: Insufficient sanitization of input when using Object-Graph 
```

Comment:
This reads like an NVD description. Please put it in more laymen's terms. Remember, the target audience doesn't know anything about security.

Reply:
I have added to the description to help elaborate, hopefully addressing this comment.

Reply:
On the first line of the new description, you might want to change "paramters" to "parameters". Additionally, could you expand on the OGNL acronym to make it easier for someone unfamiliar with Struts to read? Other than that, I think the new description is pretty good.

Reply:
fixed